IgdbApiWrapper.js

```diff
@@ -13,9 +14,77 @@ function IgdbWrapper(){
 };
 
 IgdbWrapper.prototype.getGamesData = async function(){
-    let response = await request.get(this.options);
+    const gamesEndpointOptions = this.makeEndpointOptions({
+        path: 'games/',
+        queryParams: {
+            fields: ['name', 'popularity', 'cover.url', 'id'],
+            order: 'popularity:desc'    
+        }
+      }
+    );
+    let response = await request.get(gamesEndpointOptions);
     let gamesData = JSON.parse(response);
     return gamesData;    
 };
+IgdbWrapper.prototype.getGameData = async function(gameId){
+    const gameEndpointOptions = this.makeEndpointOptions({
+        path: `${"games/" + gameId}`,
+        queryParams: {
+            fields: ['name', 'summary', 'first_release_date', 'platforms',
+                'developers', 'websites', 'popularity', 'cover.cloudinary_id']
+        }
+    });
+    let response = await request.get(gameEndpointOptions);
+    let gameData = JSON.parse(response);
+    return gameData;
+};
+IgdbWrapper.prototype.makeEndpointOptions = function(endpointParams){
+    let endpointOptions = Object.assign({}, this.options);
+    endpointOptions.url = buildUrl(this.options.url, endpointParams);
+    return endpointOptions;
+};
+IgdbWrapper.prototype.getPlatforms = async function(platforms){
+    const platformEndpointOptions = this.makeEndpointOptions({
+        path: `${"platforms/" + platforms.join(",")}`,
+        queryParams: {
+            fields: ['name'],   
+            }
+        }
+    );
+    let response = await request.get(platformEndpointOptions);
+    return JSON.parse(response);
+};
+IgdbWrapper.prototype.getWebsites = function(websites){
+    const categories = {
+        "1": "official",
+        "2": "wikia",
+        "3":	"wikipedia",
+        "4":	"facebook",
+        "5":	"twitter",
+        "6":	"twitch",
+        "8":	"instagram",
+        "9":	"youtube",
+        "10":	"iphone",
+        "11":	"ipad",
+        "12":	"android",
+        "13":	"steam",
+        "14":	"Reddit"
+    };
+    let websitesData = websites.map((website)=>{
```

Comment:
Put spaces before and after arrow when defining an arrow function. Also you can omit parentheses when you have one argument
```suggestion
    let websitesData = websites.map(website => {
```